specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json

```diff
@@ -0,0 +1,932 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Local Search API",
```

Comment:
Please use "Local Search Client".
title will be used as client class name.

Reply:
DONE. See 
https://github.com/Azure/azure-rest-api-specs/pull/4349/commits/8ddfa8da1b8910b62f375dab48b937f5233bc4f6